app/questionPage.html

```diff
@@ -132,36 +132,11 @@ <h4 class="timer"></h4>
         </div>
         <div class="result_card">
             <div class="greetings">
-                <p>Your score</p>
+                <p>Thank You!</p>
             </div>
-            <div class="result">
-                <div class="answered col-xs-6">
-                    <div class="answered_label">
-                        <h3>Answered</h3>
-                    </div>
-                    <div class="answered_number">
-                        <p><span class="answered_questions"></span></p>
-                    </div>
-                </div>
-                <div class="points col-xs-6">
-                    <div class="points_label">
-                        <h3>Points</h3>
-                    </div>
-                    <div class="points_number">
-                        <p><span class="score"></span></p>
-                    </div>
-                </div>
-            </div>
-            <div class="result">
-                <div class="percentage col-xs-12">
-                    <div class="percentage_label">
-                        <h3>Percentage</h3>
-                    </div>
-                    <div class="percentage_number">
-                        <p><span class="score_percentage"></span>%</p>
-                    </div>
-                </div>
-
+            <div class="email_for_results">
+                <h1>You have completed the test!</h1>
+                <h3>Please contact the office if you would like to find out your results</h3>
```

Comment:
Could you also add an email link into this message? Just to hello@mayden.academy for now, we can work out which address we actually want later on!